clients/lightrider/lightrider-ui-features.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* global ReportUIFeatures */
+
+/**
+ * Extends ReportUIFeatures to get html from a browserified ReportGenerator.
```

Comment:
```suggestion
 * Extends ReportUIFeatures for a Lightrider viewer to get html from a browserified ReportGenerator.
```
this might be a better place to mention LR